server/fleet/software.go

```diff
@@ -147,5 +147,6 @@ type VulnerabilitySource int
 
 const (
 	NVDSource VulnerabilitySource = iota
-	OVALSource
+	UbuntuOVALSource
```

Comment:
This change should be fine since we are not using the `source` column in the `software_cve` table yet. Adding different sources per platform would allow us have different presentation options per source (for example, links that don't go to the NVD website based on the source).